lib/rules/no-v-html.js

```diff
@@ -14,16 +15,32 @@ module.exports = {
       url: 'https://eslint.vuejs.org/rules/no-v-html.html'
     },
     fixable: null,
-    schema: [],
+    schema: [{
+      type: 'object',
+      properties: {
+        ignorePattern: {
```

Comment:
Please document this option in the rule docs.

Reply:
I think an example would indeed be nice, because that would make clear what sort of expressions the pattern matches against.